vllm/worker/model_runner.py

```diff
@@ -618,10 +638,19 @@ def profile_run(self) -> None:
         # Profile memory usage with max_num_sequences sequences and the total
         # number of tokens equal to max_num_batched_tokens.
         seqs: List[SequenceGroupMetadata] = []
+        # For worker that may spend additional GPU memory in vision
+        # encoding.
+        # Each request should have at least `image_feature_size` tokens.
+        if self.vision_language_config:
+            max_num_seqs = min(
```

Comment:
I currently don't understand this part, can you make the comments above clearer? (and also move them into the "if" condition) :)

Reply:
rephrased. ptal.

Reply:
Perhaps add a warning here since this will be "overriding" user configurations.